src/metabase/util/malli/describe.clj

```diff
@@ -29,6 +29,20 @@
       max (str " with length >= " max)
       :else "")))
 
+(defn pluralize-times [n]
+  (when n
+    (if (= 1 n) "time" "times")))
+
+(defn- repeat-suffix [schema]
```

Comment:
not related but I wonder if it's better to write these suffixes with defmethod? will makes adding new stuff easier.

Reply:
That's probably worth it. There might not be any more suffixes to add 🤞 -- If I touch this file again I'll do that